### PR TITLE
updated for Raster behavior change

### PR DESCRIPTION
**Pull request by ghost**
Raster had a behavior change where an instance can only belong to one RasterLayer. I've updated the sample to account for this.

### CODE REVIEW

**Comment by ghost** [comment]
@john0005 Can you please review?

File: src/main/java/com/esri/samples/raster/blend_renderer/BlendRendererController.java [hunk 1]
```diff
@@ -32,6 +28,10 @@
 import com.esri.arcgisruntime.raster.Raster;
 import com.esri.arcgisruntime.raster.SlopeType;
 
+import javafx.fxml.FXML;
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/blend_renderer/BlendRendererController.java]: Formatting for imports it off. `com.esri` should be at the bottom.

**Comment by john0005** [comment]
@tyle8552 A small formatting issue but everything else is good.

**Comment by tschie** [comment]
Fixed the imports.

**Pull request merged**